homeassistant/components/surepetcare/strings.json

```diff
@@ -6,6 +6,13 @@
           "username": "[%key:common::config_flow::data::username%]",
           "password": "[%key:common::config_flow::data::password%]"
         }
+      },
+      "reauth_confirm": {
+        "title": "[%key:common::config_flow::title::reauth%]",
+        "description": "Re-authenticate by entering password for {username}",
```

Comment:
The integration doesn't pass along description placeholders, so "username" won't be filled